meson.build

```diff
@@ -578,6 +578,18 @@ if not libzip_dep.found()
   libzip_dep = libzip_proj.get_variable('libzip_dep')
 endif
 
+# handle zstd dependency
+r = run_command(py3_exe, check_meson_subproject_py, 'zstd', check: false)
+if r.returncode() == 1 and get_option('subprojects_check')
+  error(subproject_clean_error_msg)
+endif
+
+zstd_proj = subproject('zstd', default_options: [
+    'default_library=static',
+    'static_runtime=@0@'.format(is_static_build),
+])
+libzstd_dep = zstd_proj.get_variable('libzstd_dep')
+
```

Comment:
I have not checked properly, but I saw `libzip` (which we already use in other places) have some references to zstd. Have you explored whether we can use that instead?

Reply:
<img width="835" alt="image" src="https://github.com/rizinorg/rizin/assets/24983369/f3cfdbf6-9d70-4d72-bc25-bd053bacea5c">
But it looks like libzip also uses zstd as a dependency?

Reply:
@ret2libc I think it doesn't expose "raw" ZStd API, just only when part of the ZIP archive